app/components/course/CourseCard.tsx

```diff
@@ -58,28 +16,33 @@ const CourseCard = ({
     HI: false,
     major_restricted: false
   },
+  description = "",
   term = undefined,
   prereqs = undefined,
   status = "No Selection"
 }: ICourseSchema) => {
   const [state, setState] = useState(status);
+  const courseFull = { title, courseCode, subject, attributes, description, term, prereqs, status };
   const {courses, updateCourseState} = useAppContext();
+  const [popupShown, setPopupShown] = useState(false);
   status = courses.find(course => course.title === title)?.status || "No Selection";
-  const offeredSemesters = offeredSemestersChecker(term);
+
+  const popupOpen = () => {
+    setPopupShown(true);
+  };
+
   return (
-    <section className="course-card">
+    <section className="course-card relative">
+      <div className="w-full h-full absolute hover:cursor-pointer z-0" onClick={popupOpen}></div>
```

Comment:
Minor thing but this on click is going beyond the bounds of the course card. Like you are able to click the whitespace between the course cards and it will select a card.